ansible_collections/arista/avd/roles/eos_designs/doc/connected-endpoints.md

```diff
@@ -264,8 +228,75 @@ port_profiles:
           short_esi: < 0000:0000:0000 | auto >
 ```
 
+### Network Ports
+
+```yaml
+# Network Ports | Optional
+# All switch_ports ranges are exanded into indidual port configurations.
+# Switches are matched with regex matching the full hostname.
+network_ports:
+  - switches:
+      - < regex matching the full hostname of one or more switches >
+    # Switch_ports is a list of ranges using AVD Range syntax.
```

Comment:
Do we document this range syntax anywhere? I put a few examples in my auto_interface code but it might be nice to have something central.

Reply:
there is this bit in the doc: https://avd.sh/en/latest/plugins/index.html#range_expand-filter

Reply:
added a link to the range_expand filter